.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+<!--
+Thanks for contributing to _fastlane_! Before you submit your pull request, note that files in the docs folder covering Actions (actions.md and actions/*) and Plugins (available-plugins.md) are auto-generated. Those should be edited at their sources (https://github.com/fastlane/fastlane or plugin repository) and will be updated here regularly.
+If this is the case, please modify the documentation in the actual classes from https://github.com/fastlane/fastlane.
```

Comment:
This is now duplicated in the previous sentence I think. This one sounds better, but is missing the plugin repo bit and that it will be updated regularly.

Reply:
Right, sorry about this!